lib/screens/createtrip.dart

```diff
@@ -21,9 +22,15 @@ class _CreateTripState extends State<CreateTrip> {
   DateTime _selectedEndDate;
   TimeOfDay _selectedEndTime;
   bool privacy = false;
+  final RequestService _request = RequestService();
 
-  void _addNewRequest() {
+  void _addNewRequest() async {
     final newRq = RequestDetails(name: 'Name', id: DateTime.now().toString(), destination: _destination, finalDestination: _finalDestinationController.text, startDate: _selectedStartDate, startTime: _selectedStartTime, endDate: _selectedEndDate, endTime: _selectedEndTime, privacy: privacy);
+    try {
+      await _request.createTrip(newRq);
+    } catch (e) {
+      print(e.toString());
```

Comment:
Also show the error to user using 
See dashboard.dark that how error can be shown using snack bar if _scaffoldKey is used
```dart
String errStr = err.message ?? err.toString();
final snackBar = SnackBar(content: Text(errStr), duration: Duration(seconds: 3));
_scaffoldKey.currentState.showSnackBar(snackBar);
```

Reply:
fixed the database models with the specified entries and flow. Unable to implement snackbar right now but that can be done later. will push soon.